src/Illuminate/Database/Query/Builder.php

```diff
@@ -226,10 +227,7 @@ public function __construct(ConnectionInterface $connection,
     }
 
     /**
-     * Set the columns to be selected.
-     *
-     * @param  array|mixed  $columns
-     * @return $this
+     *  {@inheritdoc}
```

Comment:
All of these docblocks have two spaces in front of `{@inheritdoc}` while there should be one. Please adjust this for all of these docblocks.

Reply:
OK, I wasn't sure since it's double-spaced in other parts of the framework. I can change it, though.

Reply:
I indeed see two usages in the container that are incorrect.